runtime/compilers/rillv1/parse_metrics_view.go

```diff
@@ -186,10 +186,14 @@ func (p *Parser) parseMetricsView(ctx context.Context, node *Node) error {
 	spec.DefaultTimeRange = tmp.DefaultTimeRange
 	spec.AvailableTimeZones = tmp.AvailableTimeZones
 
+	// create a map of dimensions to be used for policy validation
+	dimensions := map[string]bool{}
```

Comment:
There are already `names` and `columns` maps available in scope – can we find a way to just use one of those?

Reply:
ok missed that, will use